### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by wrathwolf**
Cleaning up meeting invite

### CODE REVIEW

**Comment by jpengar** [comment]
The changes look fine to me, but I think all https://github.com/camaraproject subprojects actually have the existing README structure by default. It sort of keeps consistency between them... Should we change it? For me it is fine either way, it is more of a "cosmetic" thing.

**Review by AxelNennker — CHANGES_REQUESTED** [review]
I think writing "CET/CEST" is self-contradicting, because a time is either CET or CEST, right? I can't be both at the same time, because CET is without DST, while CEST is with DST. In winter the time is the same, but the spec is either or, I think
I think, the text should be 16:00 CEST, but I am not sure about that.

File: README.md [hunk 1]
```diff
@@ -13,19 +14,13 @@ Repository to describe, develop, document and test the Identity And Consent Mana
...
-## Status and released versions
-* Release 0.1.0 of guidelines and documentation for CAMARA APIs is available within the [release-0.1.0 branch](https://github.com/camaraproject/IdentityAndConsentManagement/tree/release-0.1.0)
-
-For changes see [CHANGELOG.md](https://github.com/camaraproject/IdentityAndConsentManagement/blob/main/CHANGELOG.md)
-
-## Contributorship and mailing list
-* To subscribe / unsubscribe to the mailing list of this Sub Project and thus be / resign as Contributor please visit <https://lists.camaraproject.org/g/sp-icm>.
-* A message to all Contributors of this Sub Project can be sent using <sp-icm@lists.camaraproject.org>.
 
+## Contributing
```
Comment (hdamker) [inline README.md:18]: I propose to keep the section "Status and released versions", see also https://github.com/camaraproject/Template_Lead_Repository/pull/3

```suggestion
## Status and released versions

* The latest public release of guidelines and documentation for CAMARA APIs is available [here](https://github.com/camaraproject/IdentityAndConsentManagement/releases/latest).
* For previous releases, pre-releases and changes see [CHANGELOG.md](https://github.com/camaraproject/IdentityAndConsentManagement/blob/main/CHANGELOG.md)

## Contributing
```

**Comment by hdamker** [comment]
> The changes look fine to me, but I think all https://github.com/camaraproject subprojects actually have the existing README structure by default. It sort of keeps consistency between them... Should we change it? For me it is fine either way, it is more of a "cosmetic" thing.

@jpengar it's part of the initiative to consolidate the information, reduce duplication and make the README.md files usable for inclusion within the sub project wiki "front" page. So the idea is to make all README.md consistent to a new format.

@wrathwolf I propose that we first conclude the [PR #3](https://github.com/camaraproject/Template_Lead_Repository/pull/3) on the template, before a roll-out to the sub projects.

**Comment by hdamker** [comment]
> I think writing "CET/CEST" is self-contradicting, because a time is either CET or CEST, right? I can't be both at the same time, because CET is without DST, while CEST is with DST. In winter the time is the same, but the spec is either or, I think I think, the text should be 16:00 CEST, but I am not sure about that.

@AxelNennker The logic is that the timezone in EU countries is either CET or CEST (and never both at the same time), and in both cases the meeting is at 16:00.  The meeting setup is currently anchored in EU, so it will change automatically with the change to DST in EU (hence the remark about the UTC change if in EU is DT). 

If this is the right approach for a global project where this impacts most of the world population which does not have DST (anymore) and especially the southern hemisphere is another question which I will raise as an issue for discussion in Governance.

**Comment by hdamker** [comment]
Please comment in https://github.com/camaraproject/Template_Lead_Repository/pull/3 regarding the upcoming structure for the README.md. That should be agreed before we touch/change all repositories.

**Comment by hdamker** [comment]
@wrathwolf @jpengar @AxelNennker with the changes https://github.com/camaraproject/IdentityAndConsentManagement/pull/144/commits/ecab84486736d9aa260a0d58d13455f309bb5d7f I aligned it with [Template_Lead_Repository/blob/main/README.md](https://github.com/camaraproject/Template_Lead_Repository/blob/main/README.md).

Please review and merge if ok.

**Review by AxelNennker — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (hdamker) [reply README.md:28]: Sorry, this suggestion I had already within the my last commits, it is now doubled. Will delete it.

**Comment by hdamker** [comment]
I had to delete the doubled section with "Release Information".
BTW: feel free to suggest changes to this section if it does not fit for ICM.

**Comment by AxelNennker** [comment]
I think the following is fitting for API subprojects but not for ICM
"Service APIs for “IdentityAndConsentManagement” (see [APIBacklog.md](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/APIBacklog.md))"

Find it strange to view e.g. OIDC as a service API.

**Comment by jpengar** [comment]
> I think the following is fitting for API subprojects but not for ICM "Service APIs for “IdentityAndConsentManagement” (see [APIBacklog.md](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/APIBacklog.md))"
> 
> Find it strange to view e.g. OIDC as a service API.

+1. APIBacklog.md link is also broken. 
We may need to revise the scope section a bit for ICM. It also mentions "Describe, develop, document and test the APIs".

**Comment by hdamker** [comment]
@AxelNennker @jpengar I corrected the link for APIBacklog.md. But please consider that this PR was opened by @wrathwolf to execute on tps://github.com/camaraproject/Governance/issues/124, not to revise the content. The lines within scope you are commenting on are not even changed by this PR. I kindly ask to approve and merge this PR and then open your own issue/PR to revise the content as needed.

File: README.md [hunk 2]
```diff
@@ -4,28 +4,27 @@
...
-* To subscribe / unsubscribe to the mailing list of this Sub Project and thus be / resign as Contributor please visit <https://lists.camaraproject.org/g/sp-icm>.
-* A message to all Contributors of this Sub Project can be sent using <sp-icm@lists.camaraproject.org>.
+* The latest public release of guidelines and documentation for CAMARA APIs is available [here](https://github.com/camaraproject/IdentityAndConsentManagement/releases/latest).
+* For previous releases, pre-releases and changes see [CHANGELOG.md](https://github.com/camaraproject/IdentityAndConsentManagement/blob/main/CHANGELOG.md)
 
+## Contributing
+* Meetings
+  * Bi-weekly Wednesday, 16:00 CEST (14:00 UTC during European daylight saving time)
+  * [Registration / Join](https://zoom-lfx.platform.linuxfoundation.org/meeting/94629188836?password=278b4c8a-f370-43bf-bac1-b30a39f169f3)
+  * Minutes: Access [meeting minutes](https://wiki.camaraproject.org/x/2QBFAQ).
```
Comment (jpengar) [inline README.md:27]:
```suggestion
* Meetings
  * [Registration](https://zoom-lfx.platform.linuxfoundation.org/meeting/94629188836?password=278b4c8a-f370-43bf-bac1-b30a39f169f3)
   * Schedule: Bi-weekly Wednesday, 16:00 CEST (14:00 UTC during European daylight saving time)
  * Minutes: Access [meeting minutes](https://wiki.camaraproject.org/x/2QBFAQ).
```

**Review by jpengar — DISMISSED** [review]
LGTM. Just one suggestion to have same meeting information structure as https://github.com/camaraproject/Template_Lead_Repository/blob/main/README.md. Just in case you want to consider it.

File: README.md (hunk 2 above)
Comment (hdamker) [reply README.md]: Agreed, just that I added the "/ Join" again as you can't use the link for registration as long the meeting is active. Will change the template accordingly.

**Review by jpengar — APPROVED** [review]
LGTM

**Pull request merged**